src/object_schema.cpp

```diff
@@ -63,27 +63,27 @@ ObjectSchema::ObjectSchema(std::string name, IsEmbedded is_embedded, std::initia
     }
 }
 
-PropertyType ObjectSchema::from_core_type(Table const& table, ColKey col)
+PropertyType ObjectSchema::from_core_type(ColKey col)
```

Comment:
not having to pass around the table is a nice simplification all around the codebase 💯